client/src/components/assets/InfoCard.jsx

```diff
@@ -3,19 +3,133 @@ import CloseIcon from '@material-ui/icons/Close';
 import axios from 'axios';
```

Comment:
Low Priority (Optional for now): I wouldn't put assets in components. Normally, assets refer to image, video files.